apps/api/src/.example.env

```diff
@@ -2,6 +2,7 @@ NODE_ENV=local
 PORT=3000
 API_ROOT_URL=http://localhost:3000
 FRONT_BASE_URL=http://localhost:4200
+STORE_ENCRYPTION_KEY="<ENCRYPTION_KEY_MUST_BE_32_LONG>"
```

Comment:
Could we also add this to the `docker` folder example and pass it to the docker-compose file from the .env file?